history/petr-troyanskii.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Early machine translation scholar
+---
+# Petr Troyanskii
+Petr Troyanskii was an educator and scholar. In 1935, he filed and received USSR Patent 40995 for his invention. 
+
+The machine selected and typed words when translating from one language to another 
+or across multiple languages simultaneously.
+
+The method suggested coding and interpreting grammatical functions using universal symbols. 
+The interlingual grammatical rules were based on Esperanto. In the same years, 
+[Georges Artsrouni] (history/georges-artsrouni.md) independently filed a patent in Paris. 
+
+Troyanskii entered the University of St. Petersburg but did not finish it due to World War I. 
+Later he studied at the Institute of Red Professors. 
+
+He taught social sciences and the history of science and technology. 
+He considered mechanizing translation the cause of his whole life. 
```

Comment:
This is somewhat subjective.  If there's a quote from him or something, that would be more factual.

Reply:
I read a book chapter by Hutchins and Lovtskii "Petr Petrovich Troyanskii (1894–1950): A Forgotten Pioneer of Mechanical Translation", and it resonated with what was written in Wikipedia. I will try to reformulate this based on more factual statements.

Reply:
Thanks!

Reply:
In that case, the best way to do it is to just put a direct quote from that book.